frontend/src/index.jade

```diff
@@ -125,8 +125,10 @@ html
               span breakdown by file format
         #summary-charts
           .summary-charts(v-for="repo of filteredRepos")
-            .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoPath }}
-            .summary-charts__title(v-if="filterGroupSelection === 'groupByAuthors'") {{ repo[0].displayName }} ({{ repo[0].name }})
+            .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoPath }}&nbsp;&nbsp;
+              .group-total-contribution(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
```

Comment:
What is this `title` for?

Reply:
It is to show the info (Total contribution of group) when hovering over the number of lines instead of taking up more space and displaying together with the number of lines.

Reply:
OK, this makes sense.